mgmt/rest/v2/plugin.go

```diff
@@ -41,6 +41,15 @@ import (
 	"github.com/julienschmidt/httprouter"
 )
 
+const (
+	// TLSCertPrefix defines a prefix for file fragment carrying path to TLS certificate
+	TLSCertPrefix = "crt."
+	// TLSKeyPrefix defines a prefix for file fragment carrying path to TLS private key
+	TLSKeyPrefix = "key."
+	// TLSCACertsPrefix defines a prefix for file fragment carrying paths to TLS CA certificates
+	TLSCACertsPrefix = "cacerts."
```

Comment:
@candysmurf, could we keep one pattern and move declaration of those constants to https://github.com/intelsdi-x/snap/blob/master/mgmt/rest/v2/api.go (as it's done for V1, see https://github.com/intelsdi-x/snap/blob/master/mgmt/rest/v1/api.go)